bot/resources/evergreen/speedrun_links.json

```diff
@@ -0,0 +1,20 @@
+{
+    "links": [
+        "https://www.youtube.com/watch?v=jNE28SDXdyQ",
+        "https://www.youtube.com/watch?v=iI8Giq7zQDk",
+        "https://www.youtube.com/watch?v=VqNnkqQgFbc",
+        "https://www.youtube.com/watch?v=Gum4GI2Jr0s",
+        "https://www.youtube.com/watch?v=5YHjHzHJKkU",
+        "https://www.youtube.com/watch?v=X0pJSTy4tJI",
+        "https://www.youtube.com/watch?v=aVFq0H6D6_M",
+        "https://www.youtube.com/watch?v=1O6LuJbEbSI",
+        "https://www.youtube.com/watch?v=Bgh30BiWG58",
+        "https://www.youtube.com/watch?v=wwvgAAvhxM8",
+        "https://www.youtube.com/watch?v=0TWQr0_fi80",
+        "https://www.youtube.com/watch?v=hatqZby-0to",
+        "https://www.youtube.com/watch?v=X0pJSTy4tJI",
```

Comment:
This is a duplicate.

Reply:
Removed the duplicate punch out link and added the link you requested.